sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pandas as pd
+from gensim.models import KeyedVectors
+from sklearn.metrics import pairwise_distances
+
+
+def euclidean_similarity(vector_1: np.array, vector_2: np.array) -> np.float:
+    """
+        calculate euclidean distance between two vectors
+    """
+    return 1 / (1 + np.linalg.norm(vector_1 - vector_2))
+
+
+def cosine_similarity(vector_1: np.array, vector_2: np.array) -> np.float:
+    """
+        calculate cosine distance between two vectors
+    """
+    return np.dot(vector_1, vector_2) / (np.linalg.norm(vector_1) * np.linalg.norm(vector_2))
+
+
+def manhattan_similarity(vector_1: np.array, vector_2: np.array) -> np.float:
+    """
+        calculate manhattan distance between two vectors
+    """
+    return 1 / (1 + np.sum(np.abs(vector_1 - vector_2)))
+
+
+# def get_similarity_matrix(wv: KeyedVectors, similarity_function) -> pd.DataFrame:
```

Comment:
If we need the code in the comments, a function must be made for it. If we don't need this, just delete this comments.